### DISCUSSION TITLE
Vercel Deployed Website doesn't update blogs content through cosmicjs unless redeploy

### DISCUSSION THREAD

**Discussion by rtsuchi** [vercel/community#5323] [Help]
### Summary

When I run my project on my local development and add/update a blog using cosmicjs, it works. I can see the updated content immediately on my localhost. But the production host Https://your-perstyle.com didn't get updated. It can only updated if I manually "redeploy" it in vercel.

### Example

https://your-perstyle.com/

### Steps to Reproduce

1. Go to website https://your-perstyle.com
2. Update or Add content via CosmicJs and publish

Expected result: updated content should automatically rendered if you refresh the website
Actual result: website not updated. You have to manually "redeploy" the website via vercel.

**Comment by devIgor1** [comment]
I'm having the same issue, hope someone can help

**Comment by z4nr34l** [comment]
It's hard to give a good solution without code, but I first I would check if data is up-to-data and not cached by:

1. CMS cache
2. Data fetch cache
3. Site generation mode (Is it SSG/ISR/Dynamic)
4. Browser cache
